src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnGitRepository.cs

```diff
@@ -158,11 +149,11 @@ private async Task<AltinnStudioSettings> CreateNewAltinnStudioSettings()
             AltinnStudioSettings newAltinnStudioSettings;
```

Comment:
Long variable name, cloud it just be `settings` as we are in the context `CreateNewAltinnStudioSettings()` ?

Reply:
Yeah, you are right. I'm so used to just naming variables after the class (just with camelCase) because I've seen so many crap names that defaulting to the same as the class is an easy convention that makes sense most of the time.